iris-stubs/cube.pyi

```diff
@@ -4,23 +4,26 @@ from typing import Any, Iterable, Literal, Mapping, Optional, Tuple, TypeVar, Un
 
 from cf_units import Unit
 
+from dask.array import Array as DaskArray
+
 from iris.coords import CellMeasure, CellMethod, DimCoord, AuxCoord, Coord, AncillaryVariable, CoordExtent
 from iris.coord_systems import CoordSystem
 from iris.common import CFVariableMixin, CubeMetadata
 from iris.aux_factory import AuxCoordFactory
 from iris._constraints import Constraint
 from iris.analysis import Aggregator, Linear, Nearest, AreaWeighted, UnstructuredNearest, PointInCell
 
+from numpy import ndarray
+from numpy.ma import MaskedArray
 from numpy.typing import DTypeLike, ArrayLike
 
 
 T = TypeVar("T")
 OneOrMany = Union[T, Iterable[T]]
 StrOr = Union[str, T]  # for functions that take name or object with name
 
-# TODO: Replace with proper types
-RealData = ArrayLike
-LazyData = ArrayLike
+RealData = Union[ndarray, MaskedArray]
+LazyData = DaskArray
```

Comment:
Not sure if specific masked dask arrays are a thing?

Reply:
I don’t think they are, no